app/Http/Controllers/Backend/User/DashboardController.php

```diff
@@ -34,19 +30,16 @@ public static function getGlobalDashboard(): Paginator {
                          });
 
                          //Option 2: Status is from oneself
-                         if (auth()->check()) {
-                             $query->orWhere('users.id', auth()->user()->id);
-                         }
+                         $query->orWhere('users.id', $user->id);
 
                          //Option 3: Status is from a followed BUT not unlisted or private
-                         $query->orWhere(function(Builder $query) {
-                             $followings = Auth::check() ? auth()->user()->follows()->select('follow_id') : [];
-                             $query->whereIn('users.id', $followings)
+                         $query->orWhere(function(Builder $query) use ($user) {
+                             $query->whereIn('users.id', $user->follows()->select('follow_id'))
```

Comment:
This needs discussion since it removes the ability to access the global dashboard unauthenticated.

Reply:
Is this a Thing? I didn't know this ist possible.